package.json

```diff
@@ -96,7 +96,8 @@
 		"lint-staged": "7.3.0",
 		"lodash": "4.17.10",
 		"mkdirp": "0.5.1",
-		"node-sass": "4.11.0",
+	  	"node-sass": "4.11.0",
```

Comment:
There's mixed whitespace here. Did you add these manually? Running `npm install` reveals there are changes introduced to the branch.

Reply:
Yes, I did add those manually, as the indentation haven't had the same length. I will fix it straight away.